packages/offix-client/integration_test/package.json

```diff
@@ -3,9 +3,7 @@
   "version": "1.0.0",
   "description": "Integration tests for datasync",
   "scripts": {
-    "pretest": "kill $(lsof -t -i:4001) & node server/index.js &",
-    "test": "karma start",
-    "posttest": "kill $(lsof -t -i:4001) || true"
+    "test": "gulp test"
```

Comment:
@b1zzu is there any particular reason for adding gulp here?

Reply:
Using gulp it's possible to start the server and run the karma tests all in the same nodejs process, which means that once the tests are finished also the server will be closed. And I can also expose the server to a random port and pass it to the karma:  https://github.com/aerogear/offix/blob/41a8c1a849e657ba8dacca092a22449353f021a2/packages/offix-client/integration_test/gulpfile.js#L30 
Mine is more an experiment, we can also achieve the same result with `pretest` and `posttest` using a fixed port for the server.
Wdyt?

Reply:
I'd much prefer to go with pre and post test commands. The JavaScript community as a whole has mostly moved away from task runners like grunt and gulp as a whole because they add a lot of maintenance overhead and for little benefits. `npm` already is a perfectly good task runner that does not require any additional dependencies.

Reply:
Sounds good for me, I've reverted to npm